src/Appwrite/Platform/Workers/Messaging.php

```diff
@@ -64,24 +63,19 @@ public function action(Message $message, Database $dbForProject): void
             return;
         }
 
-
         $message = $dbForProject->getDocument('messages', $payload['messageId']);
 
-        $provider = $dbForProject->getDocument('providers', $message->getAttribute('providerId'));
-
-        $this->processMessage($dbForProject, $message, $provider);
+        $this->processMessage($dbForProject, $message);
     }
 
-    private function processMessage(Database $dbForProject, Document $message, Document $provider): void
+    private function processMessage(Database $dbForProject, Document $message): void
     {
-        $adapter = match ($provider->getAttribute('type')) {
-            'sms' => $this->sms($provider),
-            'push' => $this->push($provider),
-            'email' => $this->email($provider),
-            default => throw new Exception(Exception::PROVIDER_INCORRECT_TYPE)
-        };
+        $recipientsId = $message->getAttribute('to', []);
 
-        $recipientsId = $message->getAttribute('to');
+        if (\count($recipientsId) < 1) {
+            Console::error('Recipients not found');
+            return;
+        }
```

Comment:
Let's make sure this isn't possible at the API level and remove the check here

Reply:
resolved this